src/app/content/practiceQuestions/components/Question.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { h4Style } from '../../../components/Typography';
+import theme from '../../../theme';
+import ContentExcerpt from '../../components/ContentExcerpt';
+import { LinkedArchiveTree, LinkedArchiveTreeSection } from '../../types';
+import { PracticeAnswer, PracticeQuestion } from '../types';
+
+interface QuestionProps {
+  question: PracticeQuestion;
+  isSubmitted?: boolean;
+  showCorrect?: boolean;
+  source: LinkedArchiveTree | LinkedArchiveTreeSection;
```

Comment:
I think questions are assigned directly to the sections, not chapters so it should be only `LinkedArchiveTreeSection`